book/tables/adverse-events/aet04_pi.qmd

```diff
@@ -5,6 +5,8 @@ subtitle: Adverse Events Reported in $\geq$ 10% of Patients by Highest NCI CTCAE
 
 ------------------------------------------------------------------------
 
+{{< include ../../test_utils.qmd >}}
```

Comment:
Should this be test_save_results.qmd?

Reply:
maybe move those two files into `utils` subdir?

Reply:
moved them to `test-utils`